tendrl/gluster_integration/manager/manager.py

```diff
@@ -200,6 +266,27 @@ def on_pull(self, raw_data, cluster_id):
                     index += 1
                 except KeyError:
                     break
```

Comment:
Created https://github.com/Tendrl/gluster_integration/issues/99 to track this change in a separate PR.